app/src/features/surveys/animals/profile/captures/components/capture-card-details/components/CaptureDetails.tsx

```diff
@@ -38,7 +38,7 @@ export const CaptureDetails = (props: ICaptureDetailsProps) => {
             Capture time
           </Typography>
           <Typography color="textSecondary" variant="body2">
-            {getFormattedDate(DATE_FORMAT.MediumDateTimeFormat, captureTimestamp)}
+            {getFormattedDate(DATE_FORMAT.MediumDateTimeFormat, captureDate)}
```

Comment:
I think this needs to first combine the captureDate and captureTime. Otherwise the UI always shows `12:00 am`.